test/unit/api_spec.js

```diff
@@ -3418,6 +3418,18 @@ Caron Broadcasting, Inc., an Ohio corporation (“Lessee”).`)
 
       await loadingTask.destroy();
     });
+    it("gets text content, correctly handling documents with toUnicode cmaps that omit leading zeros on hex-encoded UTF-16", async function () {
```

Comment:
Please keep a newline between tests, so in this case above `it(...)`, similar to the ones below.